src/cmdstan-guide/optimize_config.qmd

```diff
@@ -1,23 +1,94 @@
 ---
-pagetitle: Optimization Configuration
+pagetitle: Optimization
 ---
 
-# Maximum Likelihood Estimation
-
-The `optimize` method finds the mode of the posterior distribution, assuming that there is one.
-If the posterior is not convex, there is no guarantee Stan will be able to find the global mode
-as opposed to a local optimum of log probability.
-For optimization, the mode is calculated without the Jacobian adjustment
-for constrained variables, which shifts the mode due to the change of variables.
-Thus modes correspond to modes of the model as written.
-
-The full set of configuration options available for the `optimize` method is
-reported at the beginning of the sampler output file as CSV comments.
-When the example model `bernoulli.stan` is run with `method=optimize`
-via the command line with all default arguments,
-the resulting Stan CSV file header comments show the complete set
-of default configuration options:
+# Optimization
+
+The CmdStan executable can run Stan’s optimization algorithms
```

Comment:
algorithms which -> algorithms, which

The convention is that "which" is non-restrictive and usually separated by a comma whereas "that" is restrictive.  Compare:

* The president, who lives in the White House
* The president that lives in the White House

The first assumes we know the president being talked about and are just adding some info, whereas in the second we are using the relative clause to pick the U.S. president out of a lineup of presidents.